.gitmodules

```diff
@@ -6,4 +6,8 @@
 	url = https://github.com/htacg/tidy-html5.git
 [submodule "native~/extern/enum-flags"]
 	path = native~/extern/enum-flags
-	url = https://github.com/grisumbras/enum-flags.git
+	url = https://github.com/kring/enum-flags.git
+[submodule "native~/extern/swl-variant"]
+	path = native~/extern/swl-variant
+	url = https://github.com/kring/swl-variant.git
```

Comment:
Just wanted to check, are we using these forks to avoid future updates to the original repos?

Reply:
No, we are using forks because we had to fix bugs in those two libraries. I should move them to CesiumGS at least, though I was hoping in both cases that the upstream would accept my PRs quickly and we wouldn't need to be on a fork for long.

Reply:
Here are the PRs, for reference (I only opened the first of these myself):
* https://github.com/groundswellaudio/swl-variant/pull/5
* https://github.com/grisumbras/enum-flags/pull/32